documentation/iot/master.adoc

```diff
@@ -0,0 +1,13 @@
+include::../common/attributes.adoc[]
+
+= {Title}
+
+//include::<filename>.adoc[leveloffset=+1]
+
+//include::<filename>.adoc[leveloffset=+1]
+
+
+[appendix]
+
+// Revision information
+//include::common/revision-info.adoc[]
```

Comment:
This file is probably not needed?

Reply:
The revision info file?  It's there to put a timestamp on the end of when it was last updated. It's handy.

Reply:
I was thinking the iot/master.adoc

Reply:
Oh! :)  So, this would be part of the existing guides instead of its own title?

Reply:
Creating a dedicated book for iot could also work I guess. I just saw that the assembly-iot-guide was already added to the master-openshift.adoc. I'm not sure how well it would fit in https://enmasse.io/documentation/ as we don't have 'multi-book' support there yet :)

Reply:
I'll let @jenmalloy decide when she does her review :)